### PR TITLE
Revise READMEs

### PR DESCRIPTION
**Pull request by mwichmann**
This is a project to update the README file, of which there are currently three versions.

Marked as WIP because want to do this incrementally.  So far, only the README that will be displayed on github is modified.

Expect to eventually split parts of this into a separate file (possible moving content to CONTRIBUTING).

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -128,36 +133,58 @@ Installation
...
-script as follows on Linux or UNIX::
+    # Windows variant, assuming Python Laucher:
+    C:\Users\me> py -m pip install --user scons
 
-        # python setup.py install
+    # inside a virtualenv it's safe to use bare pip:
+    $ pip install scons
 
-Or on Windows::
+    # install in a virtualenv from a wheel file:
```
Comment (mwichmann) [inline README.rst:150]: change this prompt to hint that it's a virtualenv prompt

**Comment by mwichmann** [comment]
Okay, there's still a line like this, unchanged from previous revision:
```console
$ python scripts/scons.py build/scons
```
Which, for grins, I tried:
```console
scons: Building targets ...
scons: *** Do not know how to make File target `build/scons' (/home/mats/github/scons/build/scons).  Stop.
scons: building terminated because of errors
```
So that should be updated to ... ?

Update - is this just a capitalization problem? It should be `build/SCons` ?
2nd Update - I think the target should just be removed.

File: README.rst (hunk 1 above)
Comment (mwichmann) [reply README.rst:150]: done.

File: README.rst [hunk 2]
```diff
@@ -111,11 +115,12 @@ You can execute the SCons directly from this repository. For Linux or UNIX::
 
 Or on Windows::
 
-    C:\scons>python scripts\scons.py [arguments]
+    C:\scons>py scripts\scons.py [arguments]
```
Comment (bdbaddog) [inline README.rst]: you may want to add `-3` after py..?
not sure if you can tell the launcher any version >= 3.6?

File: README.rst [hunk 3]
```diff
@@ -128,36 +133,58 @@ Installation
     Changes`_ and `Testing`_ below if you just want to submit a bug fix or
     some new functionality.
 
-Assuming your system satisfies the installation requirements in the previous
-section, install SCons from this package by first populating the build/scons/
-subdirectory.  (For an easier way to install SCons, without having to populate
-this directory, use the scons-{version}.tar.gz or scons-{version}.zip
-package.)
+The preferred way to install SCons is through the Python installer, ``pip``
+(or equivalent alternatives, such as the Anaconda installer, ``conda``).
```
Comment (bdbaddog) [inline README.rst:129]: in a virtualenv..

File: README.rst [hunk 4]
```diff
@@ -183,7 +210,7 @@ like so::
     $ chmod 755 scripts/scons.py
```
Comment (bdbaddog) [inline README.rst]: shouldn't need to chmod scons.py.. it should have executable bit set in git..?

File: README.rst [hunk 5]
```diff
@@ -561,7 +586,7 @@ to file a new bug or to add yourself to the CC list for an existing bug
 You can explore the list of existing bugs, which may include workarounds
 for the problem you've run into on GitHub Issues:
 
-        https://github.com/SCons/scons/issues
+        httpss://github.com/SCons/scons/issues
```
Comment (bdbaddog) [inline README.rst]: typo httpss

**Comment by bdbaddog** [comment]
I'm thinking we shouldn't have so much content on using plain pip, but rather just cover virtualenv installs here and then use pip install -e .  for a virtualenv they want to use to build their own code that eliminates the need for -C's...

File: README.rst (hunk 2 above)
Comment (mwichmann) [reply README.rst]: `py -3` (or indeed, without any version specifier) takes the latest installed, unless you've manually configured a default (in a `py.ini`) or a virtualenv is active (uses that one).

File: README.rst (hunk 5 above)
Comment (mwichmann) [reply README.rst]: yeah, must have changed one too many, getting the "http" from one that was already "https".

File: README.rst (hunk 4 above)
Comment (mwichmann) [reply README.rst]: shrug... that's an existing bit. I can take it out.

File: README.rst (hunk 3 above)
Comment (mwichmann) [reply README.rst:129]: I don't think we should over-push the virtualenv thing.  Unless someone is developing a Python project, it doesn't add that much value to isolate things... a "system" type install of SCons works just fine for very many cases.  Now if they need to install Python packages to work with whatever it is, the story of course changes.

File: README.rst (hunk 4 above)
Comment (bdbaddog) [reply README.rst]: yup. you can remove it.  A vanilla scripts/scons.py git clone should have +x.. Do you see that?

**Pull request merged**